server/src/domains/auth/dto/signin.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { IsEmail, IsNotEmpty, Length } from "class-validator"
+
+export class SignInDto {
+    @IsNotEmpty({ message: 'The email field cannot be empty' })
```

Comment:
Class validator generates pretty good validation comments by itself, I don't see specific cases when these are not ok

```json
{
    "statusCode": 400,
    "message": [
        "email must be an email",
        "email should not be empty",
        "password must be longer than or equal to 6 characters",
        "password should not be empty"
    ],
    "error": "Bad Request"
}
```

So I propose not to add custom messages at all.